package.json

```diff
@@ -0,0 +1,585 @@
+{
+  "name": "awesome-books-app",
+  "version": "1.0.0",
+  "description": "Html &amp; Js",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] Your project doesn't start with the command `npm run start`.  Please make sure you have added the right scripts into your package.json which are usually return  by using 'npx create-react-app' command.

Reply:
@tugsem please elaborate on the review because i honestly don't  understand what i am supposed to do, and if i submit again I will have no reveiws left. This command 'npx create-react-app' you said is above me for now because we havent been thought any thing about react so i dont know what to do

Reply:
@tugsem You are talking about react and i am just starting week 1 of module and i know nothing about react or this 'npx create-react-app' command

Reply:
@tugsem I would appreciate a reply or a clarification because i dont understand what a react command has to do with vanilla js

Reply:
@tugsem Please reply and clarify your review, you are asking 'npx create-react-app' on a vanilla js project and I think its not right

Reply:
@shegy28 I am really sorry, my mistake. You can now request another review for approval. In any troubles  occurs with the next review you can ask for second opinion or reach to student success.